cfgov/jinja2/v1/_includes/organisms/filterable-list-controls.html

```diff
@@ -181,7 +181,7 @@
 
 {% macro render(controls, index) %}
     <div class="o-filterable-list-controls"
-         id="o-filterable-list-controls">
+         id="o-filterable-list-controls-{{ index | string }}">
```

Comment:
@jimmynotjim The pagination on L:228 is passed the same fragment identifier with this index set. Is there some other reason this would not work? As is there will be problems if there are ever two instances of filterable-list-controls on the same page. Note Kurt and I found https://github.com/cfpb/cfgov-refresh/issues/2199 when testing, but that is an existing issue.